packages/fiori/test/samples/UploadCollection.sample.html

```diff
@@ -100,6 +100,10 @@ <h3>UploadCollection</h3>
 					}
 				});
 			});
+
```

Comment:
This code is executed by the browser. You should add it inside the underneath xmp tag in order to show it in the playground.

Moreover, the code here should be written in ES5, but the one that it inside the xmp tag should be ES6